app/controllers/helpers_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class HelpersController < ApplicationController
+  before_action :set_initiative, only: %i[index create destroy]
+
+  def index
+    render json: @initiative.helpers
+  end
+
+  def create
+    new_helpers = @initiative.helpers.concat helpers_from_body
+    if @initiative.update(helpers: new_helpers)
```

Comment:
This is redundant. the modification of the array on the instance is enough to update it you just need to save it after. what you want to do here is
```
@initiative.helpers << helpers_from_body
if @initiative.save
...
```

Reply:
You're right, done!